docs/installing_deis/aws.rst

```diff
@@ -140,6 +140,23 @@ For example, if your VPC has ID ``vpc-a26218bf`` and consists of the subnets ``s
     export VPC_SUBNETS=subnet-04d7f942,subnet-2b03ab7f
     export VPC_ZONES=us-east-1b,us-east-1c
 
+If you have set up private subnets in which you'd like to run your Deis hosts, and public subnets
+for the ELB, you should export the following environment variables instead:
+
+ - ``VPC_ID``
+ - ``VPC_EC2_SUBNETS``
+ - ``VPC_ELB_SUBNETS``
+
+For example, if you have a public subnet ``subnet-8cd457b3`` for the ELB and a private subnet
+``subnet-8cd457b0`` for your EC2 instances (both in ``us-east-1a``) you would export:
+
+.. code-block:: console
+
+    export VPC_ID=vpc-a26218bf
+    export VPC_ELB_SUBNETS=subnet-8cd457b3
+    export VPC_EC2_SUBNETS=subnet-8cd457b0
```

Comment:
nitpick, but would VPC_PRIVATE_SUBNETS make it more clear that these subnets are private? We could eventually add in more than just EC2 instances into the subnet.